dist/index.html

```diff
@@ -1,13 +1,255 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="uk">
   <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Uber Eats</title>
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header>
+      <div class="header">
+        <a href="#" class="header__logo">
+          <img class="header__logo__img" src="./images/logo.svg" alt="Uber-eats logo">
+        </a>
+        <form action="#" class="header__delivery-form">
+          <label>
+            <select class="header__delivery-time" name="time_of_delivery">
+              <option value="ASAP" selected>ASAP</option>
+              <option value="in_an_hour">in an hour</option>
+              <option value="in_two_hours">in two hours</option>
+              <option value="need_telephone_call">need telephone call</option>
+            </select>
+          </label>
+          <span class="header__delivery-form__text">to</span>
+          <div class="header__delivery-address" >
+            <img class="header__delivery-address__img" src="./images/Group.png" alt="">
+            <label class="header__delivery-address__label">
+                <input type="text" class="header__delivery-address__input" name="address_of_delivery" placeholder="Enter address of delivery">
```

Comment:
Делай переносы для слишком длинных тегов
```suggestion
                <input
                  type="text"
                  class="header__delivery-address__input"
                  name="address_of_delivery"
                  placeholder="Enter address of delivery"
                >
```